README.md

```diff
@@ -275,6 +283,16 @@ Additionally, the following properties also exist for Node.js functions:
   A list of Node modules that reference other files with a dynamic expression (e.g. `require(someFunction())` as opposed
   to `require('./some-file')`). This is an array containing the module names.
 
+- `runtimeAPIVersion` `number | undefined`
+
+  When a function with the new API will be detected this is set to `2`. Otherwise it is set to `1`. `undefined` is only
+  returned in case of an error.
+
+- `runtimeVersion` `string | undefined`
+
+  Which exact runtime this function should be using. (e.g. `nodejs18.x`). This value is detected based on the input
+  `nodeVersion`.
+
```

Comment:
These are the only two actual changes in this file, the rest is formatting from prettier.

`runtimeVersion` was introduced in #1367, but never documented